widget/popup_menu.go

```diff
@@ -134,7 +134,7 @@ func NewPopUpMenuAtPosition(menu *fyne.Menu, c fyne.Canvas, pos fyne.Position) *
 // NewPopUpMenu creates a PopUp widget populated with menu items from the passed menu structure.
 // It will automatically be shown as an overlay on the specified canvas.
 //
-// Deprecated: Use ShowPopUpMenuAtPosition instead.
+// Deprecated: This will no longer show the pop-up in 2.0. Use ShowPopUpMenuAtPosition() instead.
```

Comment:
There is not any `ShowPopUpMenu()`. Should we add that or should this be deprecated in favour of `NewPopUpMenuAtPosition()` with an empty position?

Reply:
I assume, you mean “`ShowPopUpMenuAtPosition`with an empty position”.
IMO, we don’t need convenience for this one.
Why would we need it?
What is special about showing a (non modal) pop-up implicitly at the top-left corner of the screen?

I think, if someone needs this, he can simply be explicit.

Reply:
You are right. I added that text. I figured it would be good as we have `ShowPopup()` and that isn't deprecated. Should that be deprecated?

Reply:
Well, I would say yes but I don’t want to be the one to decide :).

Reply:
ShowPopUp and ShowModalPopUp were not deprecated as their return types are not expose and so will not break with the upcoming changes.
If you think we should remove the functionality because it's inadvisable that's probably another PR / issue :)
That said it's too late to add a new deprecation before 2.0 so if you mark deprecated now it's probably slated for removal in 3.0 - maybe we should hang back and see how this release goes?